en/asset-workflow/sprite.md

```diff
@@ -85,29 +85,38 @@ When the original size of Texture does not match the size of the texture image m
 
   > **Note**: the Trilinear filtering in the current engine version is consistent with the Bilinear filtering effect.
 
-## genMipmaps
+### genMipmaps
 
 Intended to increase rendering speed and reduce aliasing artifacts, images are processed into sequences which consist of a series of precomputed and optimized textures, called mipmaps.
 
 Each bitmap image of the mipmap set is a downsized duplicate of the main texture, but at a certain reduced level of detail. A high-resolution mipmap image is used for high-density samples, such as for objects close to the camera. Lower-resolution images are used as the object appears farther away.
 
 When texture filtering is set to Trilinear filtering, interpolation occurs between the two nearest levels. The rendering speed increases because the mipmap image is smaller than the original when rendering remote objects, which improves the cache hit ratio in the video card sampling process. At the same time, due to the low precision of the small image of mipmap, the Moire striation is reduced, which can anti-aliased on the picture. Also because some extra textures are generated, mipmap requires about one-third of the additional memory space.
 
-## packable
+### packable
 
 If the engine enable [Dynamic Atlas](../advanced-topics/dynamic-atlas.md), dynamic atlas automatically dynamically merge the appropriate textures in the beginning scene into a larger texture to reduce drawcall. However, merging the textures into the large texture will modify the UV coordinates of the original texture, if the UV coordinate of the texture is used in the custom effect, the UV calculation in effect will be incorrect, and the packable property of the texture needs to be set to **false** to prevent the texture from being packaged into the dynamic atlas.
```

Comment:
```suggestion
If the engine has [Dynamic Atlas](../advanced-topics/dynamic-atlas.md) enabled, appropriate textures are automatically and dynamically merged in the beginning scene into a larger texture to reduce drawcalls. However, merging the textures into the large texture will modify the UV coordinates of the original texture, if the UV coordinate of the texture is used in the custom effect, the UV calculation in effect will be incorrect, and the packable property of the texture needs to be set to **false** to prevent the texture from being packaged into the dynamic atlas.
```